docs/src/dev-docs/utils/mlip.rst

```diff
@@ -0,0 +1,70 @@
+MLIP Base Classes
+=================
+
+.. automodule:: metatrain.utils.mlip
+    :members:
+    :undoc-members:
+    :show-inheritance:
```

Comment:
These should be the only lines appearing in this file. Can you please transfer the additional documentation you wrote below into the docstrings of the two base classes?